.travis.yml

```diff
@@ -18,7 +18,7 @@ matrix:
     - env: IMAGE=awslnx-systemd VERSION=2
     - env: IMAGE=azure VERSION=2
     - env: IMAGE=chrome VERSION=1
-    - env: IMAGE=dind-aws VERSION=1
+    - env: IMAGE=dind-provider-cli VERSION=1
```

Comment:
Can you just renamed the image to `dind` ?

Reply:
Fixed